### ISSUE TITLE
[CI] ConcurrentSeqNoVersioningIT failing on 7.3

### ISSUE THREAD

**Issue by matriv** [elastic/elasticsearch#46923]
Logs: https://gradle-enterprise.elastic.co/s/dudvqrc462n26/console-log?task=:server:integTest

Cannot reproduce locally with:
```
./gradlew :server:integTest --tests "org.elasticsearch.versioning.ConcurrentSeqNoVersioningIT" \
  -Dtests.seed=E55BC7C502D0CC06 \
  -Dtests.security.manager=true \
  -Dtests.locale=en-US \
  -Dtests.timezone=UTC \
  -Dcompiler.java=12 \
  -Druntime.java=8
```

Stacktrace:
```

org.elasticsearch.versioning.ConcurrentSeqNoVersioningIT > classMethod FAILED
--
java.lang.RuntimeException: file handle leaks: [FileChannel(/var/lib/jenkins/workspace/elastic+elasticsearch+7.3+periodic/server/build/testrun/integTest/temp/org.elasticsearch.versioning.ConcurrentSeqNoVersioningIT_E55BC7C502D0CC06-001/tempDir-003/node_td3/nodes/0/indices/QfF4P9FnTRuEFebjW4o-yw/2/index/_8.fdt), FileChannel(/var/lib/jenkins/workspace/elastic+elasticsearch+7.3+periodic/server/build/testrun/integTest/temp/org.elasticsearch.versioning.ConcurrentSeqNoVersioningIT_E55BC7C502D0CC06-001/tempDir-003/node_td3/nodes/0/indices/QfF4P9FnTRuEFebjW4o-yw/2/index/_8_Asserting_0.doc), FileChannel(/var/lib/jenkins/workspace/elastic+elasticsearch+7.3+periodic/server/build/testrun/integTest/temp/org.elasticsearch.versioning.ConcurrentSeqNoVersioningIT_E55BC7C502D0CC06-001/tempDir-003/node_td3/nodes/0/indices/QfF4P9FnTRuEFebjW4o-yw/2/index/_8.dim)]
 
Caused by:
java.lang.Exception
 ```

**Comment by ywelsch** [comment]
@dnhatn is this the same issue as #30290?

**Comment by dnhatn** [comment]
@ywelsch Yes, it's the same problem. We can't do anything to fix this issue on 7.3.
